test/Example01.t.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import {Test} from "forge-std/Test.sol";
+import {
+    CCIPLocalSimulator,
+    IRouterClient,
+    LinkToken,
+    BurnMintERC677Helper
+} from "@chainlink/local/src/ccip/CCIPLocalSimulator.sol";
+import {Client} from "@chainlink/contracts-ccip/src/v0.8/ccip/libraries/Client.sol";
+
+contract Example01Test is Test {
+    CCIPLocalSimulator public ccipLocalSimulator;
+
+    address alice;
+    address bob;
+    IRouterClient router;
+    uint64 destinationChainSelector;
+    BurnMintERC677Helper ccipBnMToken;
+    LinkToken linkToken;
+
+    function setUp() public {
+        ccipLocalSimulator = new CCIPLocalSimulator();
+
+        (uint64 chainSelector, IRouterClient sourceRouter,,, LinkToken link, BurnMintERC677Helper ccipBnM,) =
+            ccipLocalSimulator.configuration();
+
+        alice = makeAddr("alice");
+        bob = makeAddr("bob");
+
+        router = sourceRouter;
+        destinationChainSelector = chainSelector;
+        ccipBnMToken = ccipBnM;
+        linkToken = link;
+    }
+
```

Comment:
Could we add a brief @dev comment before this function to help explain this "arrange" stage of the test? 

Something like, alice, the sender, gets tokens to send and authorises the router to spend her tokens.